sql/src/test/java/io/crate/integrationtests/CopyIntegrationTest.java

```diff
@@ -239,16 +240,22 @@ public void testCopyFromInvalidJson() throws Exception {
     }
 
     @Test
-    public void testCopyFromFileWithPartition() throws Exception {
-        execute("create table quotes (id int, " +
-                "quote string) partitioned by (id)");
+    public void testCopyFromFileWithPartitionCreation() throws Exception {
+        // test with many shards and a small bulk size because this would trigger race conditions more reliably
+        // also use a larger data set to import
+        execute("CREATE TABLE names (group_id INTEGER, name STRING) " +
+                "PARTITIONED BY (group_id) " +
+                "CLUSTERED INTO 16 SHARDS " +
+                "WITH (number_of_replicas=0)");
         ensureGreen();
-        execute("copy quotes partition (id = 1) from ? with (shared=true)", new Object[]{
-            copyFilePath + "test_copy_from.json"});
-        refresh();
-
-        execute("select * from quotes");
-        assertEquals(3L, response.rowCount());
+        execute("COPY names PARTITION (group_id = 1) FROM ? WITH (bulk_size = 10, compression = 'gzip')",
```

Comment:
this statement sometimes takes quite a bit longer than the mean time.
that's why I added the longer timeout